.github/workflows/release.yaml

```
@@ -4,6 +4,8 @@ on:
   release:
     types: 
     - created
+  schedule:
+    - cron: '0 1 * * 1-5' # 6 P.M. PST
```

```suggestion
    - cron: '0 13 * * 1-5' # 6 P.M. PST
```

I just want to take it for a spin... we can set it back to the original time after.